firefox_ui_tests/functional/locationbar/test_star_in_autocomplete.py

```diff
@@ -45,15 +49,18 @@ def test_star_in_autocomplete(self):
         with self.marionette.using_context('content'):
             self.marionette.navigate(self.marionette.absolute_url('layout/mozilla_grants.html'))
```

Comment:
I would propose you add a `self.test_urls` to setUp() which prepares all that including the call to absolute_url(). See Matt's PR here: https://github.com/mozilla/firefox-ui-tests/pull/70/files#diff-aa5ddb89f02abb509ef9bdd35d53a9a8R25